converters/toZigbee.js

```diff
@@ -968,11 +968,17 @@ const converters = {
     thermostat_occupied_heating_setpoint: {
         key: ['occupied_heating_setpoint'],
         convertSet: async (entity, key, value, meta) => {
-            const occupiedHeatingSetpoint = (Math.round((value * 2).toFixed(1)) / 2).toFixed(1) * 100;
```

Comment:
Changes to toZigbee shouldn't be needed, the correct endpoint (entity) will be passed when using `zigbee2mqtt/[THERMOSTAT]/heat/set`/`zigbee2mqtt/[THERMOSTAT]/cool/set` as topic

Reply:
The reports from the thermostat are working, thanks. When I try to publish a command using `zigbee2mqtt/[THERMOSTAT]/heat/set` I get `Entity '[THERMOSTAT]/heat' is unknown`

Reply:
Ah sorry, the endpoint names should be one of: https://github.com/Koenkk/zigbee2mqtt/blob/master/lib/util/utils.js#L31

Reply:
When I add the endpoint names to that file it all works perfectly (without my changes to toZigbee). Should I submit a pr to zigbee2mqtt to add the heat and water endpoints? Thanks so much for your help!

Reply:
No, I will add it.